hazelcast/src/main/java/com/hazelcast/internal/util/phonehome/PhoneHomeMetrics.java

```diff
@@ -148,13 +148,15 @@ public enum PhoneHomeMetrics {
 
     // SQL METRICS
     SQL_QUERIES_SUBMITTED("sqlqs"),
+    SQL_STREAMING_QUERIES_SUBMITTED("sqlqss"),
```

Comment:
it's rather "executed streaming queries" than "submitted streaming queries", 
because if streaming query fails for some reason we don't report it. Although I'm open for naming suggestions here

Reply:
I'm changing it as SQL_STREAMING_QUERIES_EXECUTED
how about the metric name `sqlqss`, shall we change it as well? It was requested to be `sqlqss` in the Jira ticket

Reply:
please update the ticket and a comment why the name has been changed

Reply:
ok